ulwgl_run.py

```diff
@@ -327,8 +328,16 @@ def main() -> None:  # noqa: D103
         os.environ[key] = val
 
     build_command(env, command, opts)
-    subprocess.run(command)
+    proc = subprocess.run(command)
+    return proc.returncode
 
 
 if __name__ == "__main__":
-    main()
+    try:
+        ret = main()
+    except Exception as e:  # noqa: BLE001
+        traceback.print_tb(e.__traceback__, file=sys.stderr)
+        print(f"{e}", file=sys.stderr)
```

Comment:
Because of the above line, we shouldn't have to print twice, right?

Reply:
In case of `print_tb` it was required to get the message. It was printing only the traceback. In case `print_exception` behaves differently, I will remove it.

Reply:
No, I believe this is unnecessary and I've confirmed this when using `print_exception` on a syntax error